test/content-get.js

```diff
@@ -0,0 +1,45 @@
+const request = require("request");
+const {promisify} = require('util')
+const fs = require('fs')
+const writeFile = promisify(fs.appendFile)
+
+const URLS = require('./dateValidFailed.json')
+const OUTPUT_FNAME = './test/withHTML.json'
+
+const getHTML = (url) => new Promise((resolve, reject) => {
+        request(url, (err, res, body) => err ? reject(err) : resolve(body))
+    })
+
+const htmlToFile = async () => {
+    await writeFile(OUTPUT_FNAME,'[\n', 'utf8')    
+    let delimiter = '';
+    for await (const url of URLS) {
```

Comment:
`URLS` is not async, no need for `for await`. For await is only used with async iterators